src/components/RandomAppList.js

```diff
@@ -49,11 +51,11 @@ const RandomAppList = () => {
   const [randomAppList, setRandomAppList] = useState([])
 
   useEffect(() => {
-    const list = appList.map((item) => {
-      item.randomNumber = Math.floor(Math.random() * appList.length)
-      return item
-    })
-    list.sort((a, b) => a.randomNumber - b.randomNumber)
+    const list = _.shuffle(
+      appList.map((item) => {
```

Comment:
Same here, I don't believe the map is necessary since we're just iterating through each item & returning the item (i.e. not changing anything).